### DISCUSSION TITLE
Add to Bookmark manually - Full Bookmarks management maybe!

### DISCUSSION THREAD

**Discussion by Abdulali97** [dailydotdev/daily#629] [Feature Requests]
- it will be awesome to add the functionality of adding bookmarks manually and maybe full bookmark management with categories, tags, etc 
- load card with paginator instead of keep scrolling down   
:)
![image](https://user-images.githubusercontent.com/29580490/179685747-1bebbeb8-9e1c-44bc-875b-5b1532811763.png)

**Comment by idoshamun** [comment]
By adding bookmarks manually, do you mean submitting a link and saving it as a bookmark?

**Reply by Abdulali97** [reply]
yep

**Reply by idoshamun** [reply]
Thanks for the information and suggestion. Let me get back to you on that

**Reply by idoshamun** [reply]
At the moment we decided that it is out of the scope of our product. But we might change this decision in the future.
I'll let you know if something changes
